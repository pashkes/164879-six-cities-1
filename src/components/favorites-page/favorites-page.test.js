import React from "react";
import {shallow, configure} from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import {FavoritesPage} from "./favorites-page.jsx";

configure({adapter: new Adapter()});

it(`favorites page is renders correctly`, () => {
  const favorites = shallow(<FavoritesPage cities={[]} favorites={{}} isLoading={false} loadData={jest.fn()} />);
  expect(favorites).toMatchSnapshot();
});
