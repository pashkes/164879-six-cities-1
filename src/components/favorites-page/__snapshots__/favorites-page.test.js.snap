// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`favorites page is renders correctly 1`] = `
<Layout
  pageClasses=""
>
  <main
    className="page__main page__main--favorites"
  >
    <div
      className="page__favorites-container container"
    >
      <section
        className="favorites"
      >
        <h1
          className="favorites__title"
        >
          Saved listing
        </h1>
        <ul
          className="favorites__list"
        >
          <Favorites
            cities={Array []}
            favorites={Object {}}
          />
        </ul>
      </section>
    </div>
  </main>
  <footer
    className="footer container"
  >
    <Link
      className="footer__logo-link"
      to="/"
    >
      <img
        alt="6 cities logo"
        className="footer__logo"
        height="33"
        src="img/logo.svg"
        width="64"
      />
    </Link>
  </footer>
</Layout>
`;
